### PR TITLE
Update README - Distrobox might also work on other linux environments

### PR DESCRIPTION
**Pull request by josejulio**
# Changes

Updated readme to suggest use of distrobox

**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**
Added a note about using distrobox+fedora
I tested this on SteamOS (steam deck)

### CODE REVIEW

**Review by jjasghar — CHANGES_REQUESTED** [review]
We have a ton of confusion already in this project. Putting something the _might_ work seems like a bad idea.

**Comment by josejulio** [comment]
> We have a ton of confusion already in this project. Putting something the _might_ work seems like a bad idea.

Well, it works for me - should we put a list / table of tested/tried?

**Comment by josejulio** [comment]
```
It has been reported that the fedora instruction also work with distrobox. This has been tried on:
 - SteamOS using `distrobox start fedora`
```

WDYT?

**Review by leseb — COMMENTED** [review]
If we integrate this comment I'd put a warning that this is not officially supported by the project, given that we don't have any CI coverage for this case. So words of caution but I'm not opposed to this addition.

**Comment by josejulio** [comment]
> If we integrate this comment I'd put a warning that this is not officially supported by the project, given that we don't have any CI coverage for this case. So words of caution but I'm not opposed to this addition.

I'll update the wording, thanks!

**Comment by leseb** [comment]
@josejulio any updates on this? Thanks!

**Comment by josejulio** [comment]
sorry @leseb - vacations got in the way.

I updated the note, thanks!

**Comment by josejulio** [comment]
> PRs must not contain merge commits. Please rebase instead.

done

**Comment by josejulio** [comment]
Checking test failures :eyes:

**Comment by josejulio** [comment]
Everything seems to be green now!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# InstructLab 🐶 (`ilab`)
+}# InstructLab 🐶 (`ilab`)
```
Comment (leseb) [inline README.md:1]: typo?

File: README.md [hunk 2]
```diff
@@ -93,6 +93,8 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 > **NOTE:** Python 3.12 is currently not supported, because some dependencies don't work on Python 3.12, yet.
 <!-- -->
 > **NOTE:** When installing the `ilab` CLI on macOS, you may have to run the `xcode-select --install` command, installing the required packages previously listed.
+<!-- -->
+> **NOTE:** This method is not officially supported by the project: It has been reported that the Fedora instructions also work with [distrobox](https://distrobox.it/) using `distrobox start fedora`. This has been tested by users on:  SteamOS.
```
Comment (leseb) [inline README.md:97]:
```suggestion
> **NOTE:** This method is not officially supported by the project: It has been reported that the Fedora instructions also work with [distrobox](https://distrobox.it/) using `distrobox start fedora`. This has been tested by users on: SteamOS.
```

**Review by leseb — CHANGES_REQUESTED** [review]
nits

File: README.md (hunk 1 above)
Comment (josejulio) [reply README.md:1]: I'm dealing with a new keyboard with a  small enter key :-)

```suggestion
# InstructLab 🐶 (`ilab`)
```

**Review by leseb — COMMENTED** [review]
Time to squash commits please.

**Comment by josejulio** [comment]
> Time to squash commits please.

done

**Comment by bjhargrave** [comment]
> done

I think you should only have one commit here. The PR contains some stray commits.

**Comment by josejulio** [comment]
> I think you should only have one commit here. The PR contains some stray commits.

didn't double check sorry! Looks like my squash didn't work as expected ugh - thanks!

**Comment by josejulio** [comment]
Removed the stray commits, thanks!

**Review by leseb — APPROVED** [review]

**Review by tiran — APPROVED** [review]

**Comment by leseb** [comment]
@jjasghar PTAL

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks for pushing the text @josejulio.

I am however reluctant to add text to the README for something we do not support or test. The text reads as `This method is not officially supported by the project: It has been reported that the Fedora instructions also work...`. That is not very reassuring to me if I am a user reading the docs. 

I would suggest adding this to a blog or some other medium.

**Comment by josejulio** [comment]
I understand the concerns and the reasons for not wanting to include a community based and non supported results, however, I think it would be beneficial to know what has been tried and if it worked or not. 

The instructions are focused on a  Mac and Fedora (not fully supported either!) and listing alternatives might be helpful for newcomers.

Please feel free to close if that's the consensus from maintainers, thanks!

**Pull request closed without merging**